FirebaseAuth/Sources/Backend/RPC/MultiFactor/Enroll/FIRFinalizeMFAEnrollmentResponse.m

```diff
@@ -15,15 +15,23 @@
  */
 
 #import "FirebaseAuth/Sources/Backend/RPC/MultiFactor/Enroll/FIRFinalizeMFAEnrollmentResponse.h"
-
 #import "FirebaseAuth/Sources/Backend/RPC/Proto/Phone/FIRAuthProtoFinalizeMFAPhoneResponseInfo.h"
+#import "FirebaseAuth/Sources/Backend/RPC/Proto/TOTP/FIRAuthProtoFinalizeMFATOTPEnrollmentResponseInfo.h"
 
 @implementation FIRFinalizeMFAEnrollmentResponse
 
 - (BOOL)setWithDictionary:(nonnull NSDictionary *)dictionary
                     error:(NSError *__autoreleasing _Nullable *_Nullable)error {
   _IDToken = [dictionary[@"idToken"] copy];
   _refreshToken = [dictionary[@"refreshToken"] copy];
+  if (dictionary[@"phoneAuthInfo"] != nil) {
```

Comment:
Great case of what should be an associated value enum in Swift to eliminate the nullables. No action necessary now.